relay-server/src/actors/processor.rs

```diff
@@ -1181,6 +1203,40 @@ impl EnvelopeProcessorService {
         }
     }
 
+    fn handle_process_batched_metrics(&self, message: ProcessBatchedMetrics) {
```

Comment:
Is it worth extracting metric processing into a module? We've already done this for a few cases, [see](https://github.com/getsentry/relay/tree/eac1481fe37967269658fe85516404f096f696e2/relay-server/src/actors/processor).

Reply:
Agreed. @olksdr would you like to move this afterwards as part of your refactors?

Reply:
@jan-auer  yeah, I will have a look into this, and try to refactor into the separate module.